### PR TITLE
docs: Update Universal Story to include module map

### PR DESCRIPTION
**Pull request by RicardoVaranda**
`@nguniversal/module-map-ngfactory-loader` has been release and with it, support for Lazy loading!

This PR updates the existing story to guide users how they can use this functionality.

Closes: #7706

### CODE REVIEW

**Comment by filipesilva** [comment]
@alxhub can you have a look?

File: docs/documentation/stories/universal-rendering.md [hunk 1]
```diff
@@ -127,7 +138,7 @@ Then, remove the `"polyfills"` key - those aren't needed on the server, and adju
       "root": "src",
       // Build to dist-server instead of dist. This prevents
       // client and server builds from overwriting each other.
-      "outDir": "dist-server",
+      "outDir": "dist/dist-server",
```
Comment (cyrilletuzi) [inline docs/documentation/stories/universal-rendering.md]: Bad idea to put the Universal files in the same directory as normal build, I've tried this option and it causes issues. Better stay in another directory.

Comment (RicardoVaranda) [reply docs/documentation/stories/universal-rendering.md]: Thanks for the suggestion @cyrilletuzi I'll apply those changes now 😄  Just out of curiosity what are the issues caused by having it within the dist?

Comment (cyrilletuzi) [reply docs/documentation/stories/universal-rendering.md]: I don't remember exactly, but good chances it was a problem of wrong import paths (as it's in a subdirectory) or a conflict on some building steps as files are in the same main directory.

**Comment by MarkPieszak** [comment]
Didn't see this PR in there, but added TypeScript (changes similar to these) and other fixes @vikerman requested here: https://github.com/angular/angular-cli/pull/7796

**Comment by genyklemberg** [comment]
I used this answer, it is about firebase deploy, but main logic is simple https://hackernoon.com/deploy-angular-universal-w-firebase-ad70ea2413a1

**Comment by RicardoVaranda** [comment]
Overwritten by #7803

**Pull request closed without merging**